Adding ISSUE_TEMPLATE and new pull_request  Template (#132)
PrismJS/prism-themes: .github/pull_request_template.md

> ## 🙌 Thanks For Making PR
> 
> <!--

Reviewer: People will not remove this heading. I don't want most the PRs we get start with "🙌 Thanks For Making PR", so let's move this into the comment.
```suggestion
<!--
🙌 Thank you for making this PR!

```

> Before submitting your PR, please make sure that all tests pass. 
> You can run all tests using `npm test`.
> -->
> 
> #### Checklist
> <!--
> Example how to mark a checkbox :-
> - [x] My code follows the code style of this project.
> -->
> - [ ] My code follows the code style of this project.
> - [ ] I have performed a self-review of my own code.
> - [ ] My changes generate no new warnings.
> - [ ] I have run the test cases.
> - [ ] This PR is for Adding New Theme
> 
> 

Reviewer: I think the checklist is a little too long. Also, I think it should be a comment again. A lot of people just leave these types of checklists as is.

I don't think we need to encourage people to describe their work. The titles and descriptions people choose are typically enough.

```suggestion
Before submitting your PR, please check the following:

- Your code follows the code style of this project (`npm run lint`).
- You have performed a self-review of your own code.
- You have run the test cases (`npm run test`).
-->
```

Author: hmm, I will change this.
